CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ===================
```

Comment:
Seen as this date is no in the past could you update it to a possible future release date please?